varcode/transcript_mutation_effects.py

```diff
@@ -313,7 +315,7 @@ def get_class(effect):
             return _MultipleSubstitution
     return effect.__class__
 
-variant_effect_priority_list = [
+transcript_effect_priority_list = [
```

Comment:
I'd agree re: the having a list of classes being very clear. Variant effects could have a computed property, `.priority`, which deduces from that list what their priority is.